qiskit_experiments/library/randomized_benchmarking/clifford_utils.py

```diff
@@ -91,14 +112,15 @@ def random_cliffords(
             rng = default_rng(rng)
 
         if num_qubits == 1:
-            samples = rng.integers(24, size=size)
-            return [Clifford(self.clifford_1_qubit_circuit(i), validate=False) for i in samples]
+            samples = rng.integers(cls.NUM_CLIFFORD_1_QUBIT, size=size)
+            return [Clifford(cls.clifford_1_qubit_circuit(i), validate=False) for i in samples]
         else:
             samples = rng.integers(11520, size=size)
-            return [Clifford(self.clifford_2_qubit_circuit(i), validate=False) for i in samples]
```

Comment:
11520 should be `cls.NUM_CLIFFORD_2_QUBIT` ?

Reply:
yes.